docs/release-notes/mongoid-7.4.txt

```diff
@@ -199,3 +241,93 @@ an index where it wouldn't before:
 .. code-block:: ruby
 
   Band.desc(:name).count
+
+
+Return ``0`` When Aggregating Empty Result Sets
+-----------------------------------------------
+
+Mongoid 7.4 with the ``Mongoid.broken_aggregables`` option set to ``false``
+will return ``0`` from the ``sum`` method over an empty result set, for example:
+
+.. code-block:: ruby
+
+  Product.where(impossible_condition: true).sum(:price)
+  # => 0
+
+Mongoid 7.3 and Mongoid 7.4 with the ``Mongoid.broken_aggregables`` option
+set to ``true`` (the default) returns ``nil`` in this case.
+
+.. note::
+
+  The value of ``Mongoid.broken_aggregables`` option will change to ``false``
+  by default in Mongoid 8.0.
+
+
+Correct Update Behavior When Replacing Association
+--------------------------------------------------
+
+Mongoid 7.4 with the ``Mongoid.broken_updates`` option set to ``false``
+will correctly persist an ``embeds_one`` association target it is set to nil
+and then to a non-nil value, for example:
+
+.. code-block:: ruby
+
+  class Canvas
+    include Mongoid::Document
+    
+    embeds_one :palette
+  end
+  
+  canvas.palette = palette
+  canvas.palette = nil
+  canvas.palette = palette
+
+In Mongoid 7.3 and earlier, and in 7.4 with the ``Mongoid.broken_aggregables``
+option set to ``true`` (the default), ``canvas.palette`` would be ``nil`` when
+we would expect it to be ``palette``.
+
+.. note::
+
+  The value of ``Mongoid.broken_update`` option will change to ``false``
+  by default in Mongoid 8.0.
+
+
+Correct Logical ``and`` Query Generation
+----------------------------------------
+
+Mongoid 7.4 with the ``Mongoid.broken_and`` option set to ``false``
+will preserve existing conditions when using ``and`` to add new conditions
```

Comment:
I think I understand what you're trying to say but maybe something like:
"will unnecessarily add redundant conditions when the same operator is used on the same field multiple times"
Is a little more clear? What do you think?

Reply:
Or maybe I misunderstood, broken_and does not deal with *redundant* conditions.

Reply:
I added a hopefully correct output now, does that clarify things at all?

Reply:
What language would you suggest?

Reply:
Something like this maybe? 

> Mongoid 7.4 with the ``Mongoid.broken_and`` option set to ``false`` will avoid adding redundant conditions when using ``and`` to add conditions with the same operator on the same field multiple times.

Reply:
Your language looks good to me!